docs/index.md

```diff
@@ -89,7 +89,7 @@ object Hello extends IOApp.Simple:
 ```scala mdoc:reset:silent
 //> using toolkit typelevel:default
 //> using platform native
-//> using nativeVersion 0.4.15
+//> using nativeVersion 0.4.17
```

Comment:
This is annoying but now that SN 0.5 is released users will need to explicitly pin to 0.4. Hopefully just for a short time.

Reply:
Awwwwww :(